sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleMaterial.java

```diff
@@ -41,6 +41,10 @@
 	ENDOTRACHEAL_ASPIRATE,
 	BRONCHOALVEOLAR_LAVAGE,
 	BRAIN_TISSUE,
+	ANTERIOR_NARES_SWAB,
+	OROPHARYNGEAL_ASPIRATE,
+	NASOPHARYNGEAL_ASPIRATE,
```

Comment:
This diverts from the specification (OP_ASPIRATE and NP_ASPIRATE) and is not consistent with the already existing values (NP_SWAB).